### ISSUE TITLE
makedocs fails

### ISSUE THREAD

**Issue by ufechner7** [JuliaGraphics/QML.jl#176]
The doc tests fail with the following error:
```
   ✓ Documenter
  80 dependencies successfully precompiled in 79 seconds
julia --color=yes --project=docs/ docs/make.jl
[ Info: SetupBuildDirectory: setting up build directory.
[ Info: Doctest: skipped.
[ Info: ExpandTemplates: expanding markdown templates.
[ Info: CrossReferences: building cross-references.
┌ Error: no doc found for reference '[`load`](@ref)' in src/index.md.
└ @ Documenter ~/.julia/packages/Documenter/9kOxY/src/utilities/utilities.jl:44
┌ Error: no doc found for reference '[`load`](@ref)' in src/index.md.
└ @ Documenter ~/.julia/packages/Documenter/9kOxY/src/utilities/utilities.jl:44
┌ Error: no doc found for reference '[`ListModel`](@ref)' in src/index.md.
└ @ Documenter ~/.julia/packages/Documenter/9kOxY/src/utilities/utilities.jl:44
┌ Error: no doc found for reference '[`load`](@ref)' in src/index.md.
└ @ Documenter ~/.julia/packages/Documenter/9kOxY/src/utilities/utilities.jl:44
[ Info: CheckDocument: running document checks.
[ Info: Populate: populating indices.
ERROR: LoadError: `makedocs` encountered an error [:cross_references] -- terminating build before rendering.
Stacktrace:
```

**Comment by ufechner7** [comment]
OK, the main problem is that the function `load()` does not longer exist. @barche Is it correct that it was renamed to loadqml() ?

**Comment by ufechner7** [comment]
Fixed in #178

**Comment by barche** [comment]
Thanks!
